lib/galaxy/model/__init__.py

```diff
@@ -152,7 +152,7 @@ class User( object, Dictifiable ):
     histories, credentials, and roles.
     """
     # attributes that will be accessed and returned when calling to_dict( view='collection' )
-    dict_collection_visible_keys = ( 'id', 'email', 'username' )
+    dict_collection_visible_keys = ( 'id', 'email', 'username', 'deleted', 'purged', 'active', 'last_password_change' )
```

Comment:
I'd probably remove `'purged'`, users cannot be purged at the moment as far as I know.

Also, can you please add `'deleted', 'active', 'last_password_change'` to `dict_element_visible_keys` below? `dict_collection_visible_keys` is supposed to be a subset of it.